.travis.yml

```diff
@@ -77,8 +83,10 @@ before_script:
     # Test if all new commits are signed-off
     - ./check_signed_off.sh
 
-    # Perform static code analysys
+    # Perform static code analysis
     - go tool vet -unsafeptr=false `go list ./... | grep -v Godeps | sed -e "s,^,$GOPATH/src/,;s,.*:,,"`
+    - deadcode .
```

Comment:
no, this repo has one main dir (something to keep in ming when porting to different repo)